Smartling.APITests/AuthApiClientTests.cs

```diff
@@ -6,55 +6,70 @@
 
 namespace Smartling.ApiTests
 {
-  [TestClass]
-  public class AuthApiClientTests
-  {
-    private const string ValidAuthRespone = @"{
-          ""response"" : {
-              ""code"" : ""SUCCESS"",
-              ""data"" : {
-                  ""accessToken"": ""{access token}"",
-                  ""expiresIn"": 1458,
-                  ""refreshExpiresIn"": 1455,
-                  ""refreshToken"": ""{refresh token}"",
-                  ""tokenType"": ""Bearer"",
-                  ""sessionState"": """"
-              }
+      [TestClass]
+      public class AuthApiClientTests
+      {
+        private const string ValidAuthRespone = @"{
+              ""response"" : {
+                  ""code"" : ""SUCCESS"",
+                  ""data"" : {
+                      ""accessToken"": ""{access token}"",
+                      ""expiresIn"": 1458,
+                      ""refreshExpiresIn"": 1455,
+                      ""refreshToken"": ""{refresh token}"",
+                      ""tokenType"": ""Bearer"",
+                      ""sessionState"": """"
+                  }
+          }
+          }";
+        
+        [TestMethod]
+        public void AuthenticateTest()
+        {
+          // Arrange
+          var client = new Mock<AuthApiClient>("test", "test");
+          client.CallBase = true;
+          client.Setup(foo => foo.GetResponse(It.IsAny<WebRequest>())).Returns(ValidAuthRespone);
+          client.Setup(foo => foo.PrepareJsonPostRequest(It.IsAny<string>(), It.IsAny<Object>(), It.IsAny<string>())).Returns((WebRequest) null);
+
+          // Act
+          var result = client.Object.Authenticate();
+
+          // Assert
+          Assert.AreEqual("{access token}", result.data.accessToken);
+          Assert.AreEqual("{refresh token}", result.data.refreshToken);
+        }
+
+        [TestMethod]
+        public void UserAuthenticateTest()
```

Comment:
Put your tests in separate file. 1 class file = 1 test file

Reply:
Done.